code/modules/reagents/chemistry/reagents/drink_reagents.dm

```diff
@@ -323,6 +323,83 @@
 	..()
 	. = 1
 
+/datum/reagent/consumable/tea/red
+	name = "Red Tea"
+	description = "Tasty red tea, it has digestive properties. Drink in moderation!"
+	color = "#101000" // rgb: 16, 16, 0
+	nutriment_factor = 0
+	taste_description = "sweet red tea"
+	glass_icon_state = "teaglass"
+	glass_name = "glass of red tea"
+	glass_desc = "A piping hot tea that helps with digestion of food."
+
+/datum/reagent/consumable/tea/red/on_mob_life(mob/living/carbon/M)
+	if(M.nutrition > NUTRITION_LEVEL_HUNGRY)
+		M.adjust_nutrition(-3)
+	M.dizziness = max(0,M.dizziness-2)
+	M.drowsyness = max(0,M.drowsyness-1)
+	M.jitteriness = max(0,M.jitteriness-3)
+	M.adjust_bodytemperature(23 * TEMPERATURE_DAMAGE_COEFFICIENT, 0, BODYTEMP_NORMAL)
+	. = 1
+
+/datum/reagent/consumable/tea/green
+	name = "Green Tea"
+	description = "Tasty green tea, known to heal liver it's good for you!"
+	color = "#101000" // rgb: 16, 16, 0
+	nutriment_factor = 0
+	taste_description = "tart green tea"
+	glass_icon_state = "teaglass"
+	glass_name = "glass of tea"
+	glass_desc = "A calming glass of green tea to help get you through the day."
+
+/datum/reagent/consumable/tea/green/on_mob_life(mob/living/carbon/M)
+	M.adjustOrganLoss(ORGAN_SLOT_LIVER, -0.5) //Detox!
+	M.dizziness = max(0,M.dizziness-2)
+	M.drowsyness = max(0,M.drowsyness-1)
+	M.jitteriness = max(0,M.jitteriness-3)
+	M.adjust_bodytemperature(15 * TEMPERATURE_DAMAGE_COEFFICIENT, 0, BODYTEMP_NORMAL)
+	. = 1
+
+/datum/reagent/consumable/tea/forest
+	name = "Forest Tea"
+	description = "Tea mixed with honey, has antitoxins and sweetness in one!"
+	color = "#101000" // rgb: 16, 16, 0
+	nutriment_factor = 0
+	quality = DRINK_NICE
+	taste_description = "sweet tea"
+	glass_icon_state = "teaglass"
+	glass_name = "glass of forest tea"
+	glass_desc = "A lovely glass of tea and honey."
+
+/datum/reagent/consumable/tea/forest/on_mob_life(mob/living/carbon/M)
+	if(M.getToxLoss() && prob(40))//Two anti-toxins working here
+		M.adjustToxLoss(-1, 0, TRUE) //heals TOXINLOVERs
+		//Reminder that honey heals toxin lovers
+	M.dizziness = max(0,M.dizziness-2)
+	M.drowsyness = max(0,M.drowsyness-1)
+	M.jitteriness = max(0,M.jitteriness-3)
+	M.adjust_bodytemperature(15 * TEMPERATURE_DAMAGE_COEFFICIENT, 0, BODYTEMP_NORMAL)
+	. = 1
+
+/datum/reagent/consumable/tea/mush
+	name = "Mush Tea"
+	description = "Tea mixed with mushroom hallucinogen, used for fun rides or self reflect."
```

Comment:
should be
"Tea mixed with mushroom hallucinogen, used for fun rides or self reflection."